src/api/app/components/bs_request_comment_component.html.haml

```diff
@@ -1,6 +1,7 @@
 .d-flex.mt-3.align-items-start
-  %i{ title: 'Comment', class: "fas fa-lg fa-comment #{ level > 1 ? 'd-none' : '' }" }
-  = render(AvatarComponent.new(name: comment.user.name, email: comment.user.email, size: 35, shape: :circle, custom_css: 'avatars-counter'))
+  - unless @nodecoration
```

Comment:
I don't like the idea of customizing view components using specific corner-case parameters, and the View Components documentation states that [using inheritance](https://viewcomponent.org/viewcomponents-at-github.html#avoid-inheritance) (to develop a version of this View Component for a specific case) is not recommended, so any other ideas would be very welcome...

Reply:
What is the purpose of this `@nodecoration`?

Reply:
The `@nodecoration` removes the user avatar and the username, because we already know who the user is. That way we can reclaim some space

Reply:
This are the things that are removed 
![image](https://github.com/openSUSE/open-build-service/assets/2650/4fdb5804-54f4-492a-a778-059bc5c9fc40)